collect.py

```diff
@@ -200,9 +200,21 @@ def do_something(driver, elem_properties=None):
 
 
 def screenshot(driver, file_path):
-    driver.get_screenshot_as_file(file_path)
-    image = Image.open(file_path)
-    image.save(file_path)
+    WINDOW_HEIGHT = 732
+    WINDOW_WIDTH = 412
+    page_height = driver.execute_script('return document.body.scrollHeight;')
+    page_width = driver.execute_script('return document.body.scrollWidth;')
+    height = 0
+    while height < page_height:
+        width = 0
+        while width < page_width:
+            file_name = '_'.join(file_path.split('_')[:-1] + ['H', str(width), 'V', str(height)] + file_path.split('_')[-1:])
```

Comment:
I'm not particulary fond of this, but I'll accept it for now.
We should add some code in utils to build a name and to parse a name.

Reply:
Should I create a new PR with the change?

Reply:
Let's file an issue, it's not a high priority.